services/explorer/src/components/Header/Header.tsx

```diff
@@ -12,7 +12,7 @@ import { theme } from '../../theme-provider';
 import { TaraxaIcon } from '../icons';
 import { HeaderBtn, useHeaderEffects } from './Header.effects';
 
-export const Header = () => {
+export const Header = (): JSX.Element => {
```

Comment:
Same here. If we add the type, let's add FC

Reply:
Same